.github/workflows/unittest.yml

```diff
@@ -16,6 +16,7 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
+        sudo apt-get update
```

Comment:
This causes a merge conflict. Please change it to `sudo apt-get clean`.
Don't change it for pylint.yml!

Reply:
I think changing to `sudo apt-get clean` makes unittest error again.

Reply:
Hmm, I think so too. 
Since there is no other PR in this repository, let's revert it to `sudo apt-get update` and resolve the merge conflict.
Please update your code, then I will resolve the conflict.

Reply:
Update code.

Reply:
Good, I resolved the conflict.